setup.py

```diff
@@ -31,6 +31,7 @@
         'Programming Language :: Python :: 3.5',
```

Comment:
@mehaase there is no CI coverage for 3.5

Reply:
I did see there is #112 for that and I am happy to add it into this PR to get rid of it unless you want to support it. I only need 3.6+

Reply:
yes I forgot about #112 and PR #84

PR contribution is welcome, but that should be a separate PR

Reply:
Raised #138